woocommerce/woocommerce-admin
# Render SummaryNumber delta precisely.

Previously only a given delta was rendered as an integer. Now it renders the number with a given precision.
Fixes https://github.com/woocommerce/woocommerce-admin/issues/5925.


### Accessibility

I didn't add any new interactions


### Screenshots



### Detailed test instructions:

1. Create an instance of `<SummaryNumber>` with `delta=10.12`, 
    ```js
    <SummaryNumber
		label={ "Net Sales" }
		value={ "$24.00" }
		delta={ 10.12 }
	/>
    ```
2. Check how it's rendered.

Or 
1. Build storybook `npm run storybook`
2. Open [`SummaryList` demo](http://localhost:6007/?path=/story/woocommerce-admin-components-summarylist--basic)
3. Check how the number is rendered.

![image](https://user-images.githubusercontent.com/17435/103286435-6a4f0e80-49e0-11eb-85a1-510ea2493093.png)




## Risks:
There may be places that rely on the previous behavior to format the number. Expecting `<SummaryNumber delta={ 10.0000000001 }/>` to render only `10%`.
After this change may get unexpected results. However, this behavior was not documented before, so I'd consider it was risky in the first place to rely on undocumented side-effects.

## Addtitional comments:
Previously it was truncating the number, not rounding it, which I find misleading from the user perspective.
